src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -22,19 +43,25 @@ const getAlertColorStyles = ({backgroundLight, border, iconColor}) => `
   }
 `;
 
-const getSystemAlertColorStyles = ({backgroundLight, border, iconColor}) => `
+const getSystemAlertColorStyles = ({border, iconColor}: AlertThemeProps) => css`
   border: 0;
   border-radius: 0;
-  border-bottom: 1px solid ${color(border)
-    .alpha(0.5)
-    .string()};
+  border-bottom: 1px solid
+    ${color(border)
+      .alpha(0.5)
+      .string()};
 
   ${StyledInlineSvg} {
     color: ${iconColor};
   }
 `;
 
-const alertStyles = ({theme, type, system, alignTop}) => css`
+const alertStyles = ({
+  theme,
+  type = DEFAULT_TYPE,
+  system,
+  alignTop,
+}: Props & {theme: any}) => css`
```

Comment:
@EvanPurkhiser What's the rationale for not putting `theme` in the type `Props` and just adding it here?

Reply:
The `theme` object isn't a prop that you would pass to the component. it's injected by styled itself.